patches/tinyemu/tinyemu/slirp/cksum.c

```diff
@@ -75,7 +75,7 @@ int cksum(struct mbuf *m, int len)
 	/*
 	 * Force to even boundary.
 	 */
-	if ((1 & (long) w) && (mlen > 0)) {
+	if ((1 & (uintptr_t) w) && (mlen > 0)) {
```

Comment:
Could you add an explanation comment here?

Reply:
Not my change, applied from an upstream libslirp change. The commit message is detailed enough.